types.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module '*.css' {
+  const stylesheet: CSSStyleSheet;
+  export default stylesheet;
+}
```

Comment:
@JulianCataldo Have you tried this syntax instead? which is the PR I proposed.
You can't directly return a type

Reply:
Awesome! I was fiddling with `const`, `var`, `default`,… but couldn't recall the exact pattern, even though I'd seen it somewhere before. Thx